src/utils/bulkUpdateUtils.js

```diff
@@ -23,24 +23,23 @@ export function getElevations() {
           const updatedHydrantsMap = Immutable.Map(_.keyBy(updatedHydrants, h => h.get('id')));
           store.dispatch({ type: DATA_IMPORTED, data: { hydrants: updatedHydrantsMap } });
         }
+        return `Fetched Elevation for ${needsElevation.length} hydrants`
       });
   }
+    return Promise.resolve('Hydrant Elevation Data Already Synched')
 }
 
-export function autonameHydrants(hydrants, override = false) {
-  const sortedTrailGroupedHydrants = hydrants.groupBy(h => h.get('trail'))
-    .map((trailGroup) => {
-      return trailGroup.toList().sort((h1, h2) => h1.get('elevation') - h2.get('elevation'));
-    });
-  const newHydrants = hydrants.map((h) => {
-    const trail = h.get('trail');
-    if (trail) {
-      const elevationIndex = sortedTrailGroupedHydrants.get(trail).indexOf(h);
-      return h.set('name', String(elevationIndex + 1));
-    }
-    return h;
-  });
-  return newHydrants;
+export function autonameHydrants(trail, prefix) {
+
+  const { hydrants } = store.getState().hydrants
```

Comment:
Why switch the argument from hydrants to trail? Looks like it's only used to filter the hydrants down, so can just pass in the filtered hydrant list. All functions as simple as possible!

Better not to call store directly and be more deliberate in the function arguments.